src/mscorlib/corefx/System/Globalization/TextInfo.cs

```diff
@@ -38,38 +36,28 @@ public partial class TextInfo : ICloneable, IDeserializationCallback
         ////
 
         [OptionalField(VersionAdded = 2)]
-        private String m_listSeparator;
```

Comment:
this change will be interesting to the serialization. this will cause use to manually serialize/deserialize such objects to match the fields names of the serialized objects came from the desktop